glossary.rst

```diff
@@ -0,0 +1,125 @@
+.. _glossary:
+
+========
+Glossary
+========
+
+This page contains definitions for Flux-specific terms and concepts mentioned
+throughout the various guides and docs on our sites.
+
+------------------------------------------------------------------------------
+
+.. _flux-broker:
+
+**Flux Broker**
+  A distributed message broker that provides communications services
+  within a Flux instance. It is the main controller of the Flux instance and
+  loads all the important job, scheduling, and resource related services for
+  Flux.
+
+.. _flux-instance:
+
+**Flux Instance**
+  A group of flux-broker processes which are launched via any parallel launch
+  utility that supports PMI (Process Manager Interface).
+
+.. _eventlog:
+
+**Eventlog**
+  An ordered log of timestamped events, stored in the Flux KVS and defined by
+  `RFC 18 <https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_18.html>`_.
+  Eventlogs are used to record job events, capture standard I/O streams,
+  and record resource status changes.
+
+.. _fluid:
+
+**FLUID (Flux Locally Unique ID)**
+  IDs used to represent Flux jobs, defined by
+  `RFC 19 <https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_19.html>`_.
+
+.. _hostlist:
+
+**Hostlist**
+  A format supported by existing HPC tools to create a compact list of hosts by
+  name, consisting of a common prefix, a number, and an optional suffix. For
+  example, the hostlist ``"foo[1-5]"`` can be expanded to
+  ``"foo1,foo2,foo3,foo4,foo5"``.
```

Comment:
instead of "existing", "many"?

Reply:
Perhaps say "the hosts foo1,foo2, and foo3 could be represented by the hostlist foo[1-3]."?

Reply:
Yeah I like that! And maybe go nuts and pick a fun word like fry, cow, cat, bun, fun.... there are lots of good ones! https://www.playosmo.com/kids-learning/3-letter-words-for-kids/ :laughing: 

:potato: :cow: :cat: :bread: :tada: 

Extra points for a matching emoji!  A nicely placed emoji, maybe one per page of docs, makes them more fun.

Reply:
Cool! Thanks to you both for the suggestions. I just force-pushed a change that re-worded the example at the end of the definition for hostlist. I chose `cow` for our pretend cluster, lol. 🐮